lib/bonus.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Bonus
+  attr_reader :pins
+  def initialize(num, pins)
+    @pins = pins
+  end
+
+end
```

Comment:
What is the Bonus class supposed to do?

Reply:
Sorry just finished your README and see you say you haven't finished the bonus class completely yet.